### PR TITLE
add init command

### PR DESCRIPTION
**Pull request by ralph-pichler**
add a `bee init` command which only does key generation.

### CODE REVIEW

File: cmd/bee/cmd/init.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+				logger = logging.New(cmd.OutOrStdout(), logrus.TraceLevel)
+			default:
+				return fmt.Errorf("unknown verbosity level %q", v)
+			}
+
+			_, err = c.configureSigner(cmd, logger)
+			if err != nil {
+				return err
+			}
+			return nil
```
Comment (janos) [inline cmd/bee/cmd/init.go:48]:
```suggestion
			return err
```

File: cmd/bee/cmd/start.go [hunk 2]
```diff
@@ -33,6 +33,131 @@ import (
...
+type signerConfig struct {
+	keystore         keystore.Service
+	signer           crypto.Signer
+	address          swarm.Address
+	publicKey        *ecdsa.PublicKey
+	libp2pPrivateKey *ecdsa.PrivateKey
+	password         string
+}
+
+func (c *command) configureSigner(cmd *cobra.Command, logger logging.Logger) (config *signerConfig, err error) {
```
Comment (janos) [inline cmd/bee/cmd/start.go:45]: I would put this to the end of this file or even in init.go file, but it is a very minor thing, nothing crucial, just a suggestion.

**Review by janos — APPROVED** [review]

**Comment by vandot** [comment]
When you execute `bee init` it prints log lines (with timestamps and log lvl to stdout)
@Eknirwhat do you think?

**Comment by Eknir** [comment]
> When you execute `bee init` it prints log lines (with timestamps and log lvl to stdout)
> @Eknir what do you think?

Just went over it together with you. From a product perspective, it seems to behave as it should.

**Review by acud — APPROVED** [review]

File: cmd/bee/cmd/init.go (hunk 1 above)
Comment (ralph-pichler) [reply cmd/bee/cmd/init.go]: fixed

File: cmd/bee/cmd/start.go (hunk 2 above)
Comment (ralph-pichler) [reply cmd/bee/cmd/start.go]: I just put it at the end of start.go

**Pull request merged**